cmd/envtool/main.go

```diff
@@ -219,7 +198,39 @@ func setupMonilaAndValues(ctx context.Context, pgPool *pgdb.Pool) {
 	logger.Infof("Done in %s.", time.Since(start))
 }
 
-func main() {
+func printDiagnosticData(err error) {
+	info := version.Get()
+	msg := fmt.Sprintf(`Looks like something went wrong..
+Please file an issue with all that information below:
+	Go version: %s
+	OS: %s
+	Arch: %s
+	Version: %s
+	Commit: %s
+	Branch: %s
+	Error: %v
+`,
+		runtime.Version(),
+		runtime.GOOS,
+		runtime.GOARCH,
+		info.Version,
+		info.Commit,
+		info.Branch,
+		err,
+	)
+	fmt.Println(msg)
```

Comment:
could you please remove debug log

Reply:
It's not a debug log. It prints all collected data. I think I should replace it with `fmt.Printf`.

Reply:
yep, it was just highlighted by linter that's security was not ok. thanks

Reply:
maybe change it to smth better (more safe)?

Reply:
I've added `nolint` as discussed.